spec/index.html

```diff
@@ -1362,6 +1367,26 @@ <h2>Internationalization Considerations</h2>
   <p class="issue" data-number="9"></p>
 </section>
 
+<section id="iri-abnf" class="appendix informative">
+  <h2>IRI Grammar</h2>
+
+  <p>
+    The following [[ABNF]] grammar applies the changes from [[RFC3987]] to the section
```

Comment:
```suggestion
    The following [[ABNF]] grammar applies the changes from [[RFC3987]] and [[RFC 8820]] to the section
```

Reply:
8820 is a best practice document. I don't see any grammar changes. 6874 makes a change - suggestion updated.

Reply:
In the RFCs, the grammar lines are indented by 3 spaces. Can we please keep that?

Reply:
That’s respec likely normalizing the leading space. It may take a hook to add leading white space programmatically.

Reply:
They are not in `iri-grammar.abnf`.